src/cryptonote_protocol/quorumnet.cpp

```diff
@@ -564,6 +565,12 @@ void handle_obligation_vote(Message& m, QnetState& qnet) {
     }
 }
 
+void handle_timestamp(Message& m, QnetState& qnet) {
+    MDEBUG("Received a timestamp request from " << to_hex(m.conn.pubkey()));
+    time_t seconds = time(NULL);
+    m.send_reply(get_data_as_string(seconds));
```

Comment:
I'd suggest just using `std::to_string()` here.  `get_data_as_string` is for making a string out of the underlying byte representation of an object, so here you'd get something like the 8 bytes `"\xc5\x70\xc6\x5f\x00\x00\x00\x00" to represent timestamp 1606840517 (but that value could be different on different systems).